code/datums/materials/basemats.dm

```diff
@@ -75,16 +78,17 @@
 	name = "plasma"
 	id = "plasma"
 	desc = "Isn't plasma a state of matter? Oh whatever."
-	color = "#c716b8"
+	color = "#eb80f2"
 	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/plasma
 	coin_type = /obj/item/coin/plasma
+	armor_modifiers = list("melee" = 1.4, "bullet" = 0.7, "laser" = 0, "energy" = 1.2, "bomb" = 0, "bio" = 1.2, "rad" = 1, "fire" = 0, "acid" = 0.5)
```

Comment:
it'd be funny if this had a negative fire modifier

Reply:
You'd blow up before it matters

Reply:
Still, even the plasma medal gives a little bit of fire vulnerability; I don't see why you can't just give this a -0.1 fire armor modifier or something